src/sass/_theme_layout.sass

```diff
@@ -70,8 +70,7 @@ html
       margin-left: -1.2em
       @extend .fa
       @extend .fa-plus-square-o
-      font-size: .8em
-      line-height: 1.6em
+      line-height: 18px
```

Comment:
is the change to px units needed to fix this?

Reply:
Yes because of https://github.com/readthedocs/sphinx_rtd_theme/blob/Blendify/nav-btn/src/sass/_theme_layout.sass#L169, I

Reply:
In the future, we should work on using one unit system, bootstrap uses rem, so we will likely use that in the future.